src/v1/device_manager_client.js

```diff
@@ -128,12 +128,12 @@ class DeviceManagerClient {
       devicePathTemplate: new gaxModule.PathTemplate(
         'projects/{project}/locations/{location}/registries/{registry}/devices/{device}'
       ),
-      deviceRegistryPathTemplate: new gaxModule.PathTemplate(
-        'projects/{project}/locations/{location}/registries/{registry}'
-      ),
       locationPathTemplate: new gaxModule.PathTemplate(
         'projects/{project}/locations/{location}'
       ),
+      registryPathTemplate: new gaxModule.PathTemplate(
```

Comment:
@alexander-fenster looks like changing the resource option path from `/DeviceRegistry` to `/Registry` has resulted in a breaking change. Is this intended or should we hold off on merging?

Reply:
It's a breaking change and it's being discussed within the team, let's hold off on merging for now.

Reply:
we discussed this offline in chat, a version of the library with `deviceRegistryPath` is considered a bug, as such we're releasing this as a non-breaking fix.